stylesheets/partials/features/_header-alert.scss

```diff
@@ -88,22 +89,23 @@
 
 	.alert-wrapper {
 
-		.pull-right {
-			float: none !important;
+		p > span > i {
```

Comment:
Good call here.  Exceptions to bootstrap behavior seem like something that'd be cumbersome to maintain.